src/components/plots/barPlot.ts

```diff
@@ -206,16 +206,16 @@ export module Plots {
     }
 
     /**
-     * Returns the Entity nearest to the query point according to the following algorithm:
-     *   - If the query point is inside a bar, returns the Entity for that bar.
-     *   - Otherwise, gets the nearest Entity by the primary direction (X for vertical, Y for horizontal),
+     * Returns the PlotEntity nearest to the query point according to the following algorithm:
+     *   - If the query point is inside a bar, returns the PlotEntity for that bar.
+     *   - Otherwise, gets the nearest PlotEntity by the primary direction (X for vertical, Y for horizontal),
      *     breaking ties with the secondary direction.
-     * Returns undefined if no Entity can be found.
+     * Returns undefined if no PlotEntity can be found.
      *
      * @param {Point} queryPoint
-     * @returns {Plots.Entity} The nearest Entity, or undefined if no Entity can be found.
+     * @returns {Plots.PlotEntity} The nearest PlotEntity, or undefined if no PlotEntity can be found.
```

Comment:
`Plots.PlotEntity` not in sync with the bellow where we say `PlotEntity` (not prefixed)